tests/phpunit/admin/test-class-wc-stripe-settings-controller.php

```diff
@@ -22,17 +22,12 @@ class WC_Stripe_Settings_Controller_Test extends WP_UnitTestCase {
 	public function setUp() {
 		parent::setUp();
 
-		$this->mock_account = $this->getMockBuilder( 'WC_Stripe_Account' )
```

Comment:
Was set as an instance property (but not declared) and was not accessed anywhere else - just made it a local variable instead.